astroquery/esa/hubble/core.py

```diff
@@ -215,6 +219,85 @@ def get_observation_type(self, observation_id):
             raise ValueError("Invalid Observation ID")
         return obs_type
 
+    def get_observations_from_program(self, program, *, output_file=None,
+                                      output_format="votable", verbose=False):
+        """
+        Retrieves all the observations associated to a program/proposal ID
+
+        Parameters
+        ----------
+        program : int
+            Program or Proposal ID associated to the observations
+        output_file : string
+            optional, default None
+            file name to be used to store the result
+        output_format : string
+            results format. Options are:
+            'votable': str, binary VOTable format
+            'csv': str, comma-separated values format
+        verbose : bool
+            optional, default 'False'
+            flag to display information about the process
+
+        Returns
+        -------
+        None. It downloads the artifact indicated
```

Comment:
As far as I see doesn't, it returns a Table

Reply:
Indeed, it returns a table. Copy/paste problem! It has been fixed now.